.editorconfig

```diff
@@ -7,6 +7,8 @@ indent_size = 4
 trim_trailing_whitespace = true
 [*.{phpt,php}]
 indent_style = space
+[apc.php]
+indent_style = tab
```

Comment:
Would you mind (in a separate PR) reformatting the file to spaces? We should not use tabs in PHP files.

Reply:
I split it into a separate commit so I don't get merge conflicts when I rebase, #478 has just the reformatting